### ISSUE TITLE
Providing a link or embed code to the latest version collection.

### ISSUE THREAD

**Issue by GHBulborb** [a85/POSTMan-Chrome-Extension#7619]
When updating a collection, the url link or embed code is changed because Postman is tracking versions so when saving a collection a new id is generated. This is very unfortunate when you are sharing a link to the collection on your website. When you save, it invalidates the link on the website.

A solution for this would be to provide a permalink for a collection, that always point to the latest version of a collection. Publishing this link on your website would make it valid and up-to-date always. If you want to publish an older version, use the time-based links as they are now available.

This would save so much time and hassle when you want to share postman collections with your audience. Especially when you have a large amount of collections (25+) to share.

**Comment by GHBulborb** [comment]
Would be so good if this request would make it. We would finally be able to provide our customers the Postman collections again. Because of the manual labor involved copying the new url to the CMS each time you save a collection, multiplied by the number of APIs we offer, we pulled it. But customers still would like to have the Postman collections.

**Comment by Raja-Simha** [comment]
Thank you for your feedback @GHBulborb. We are aware of the static nature of links and embedded code. We are working on providing a dynamic link to a collection that will point to the latest version of the collection. We also plan to provide a dynamic embeddable Run In Postman but cannot give you any ETA on that yet.

**Comment by camclark** [comment]
Also would be happy to see this feature implemented!

**Comment by JamesMessinger** [comment]
> We also plan to provide a dynamic embeddable Run In Postman but cannot give you any ETA on that yet.

I've discovered a workaround in the meantime that may be helpful to folks.  When you publish collection documentation using Postman, the docs page has a Run In Postman button at the top, and it **always runs the latest version** of the collection.  In your web browser's dev tools, you can inspect that button and you'll see that they're using a slightly different version of the Run In Postman embed code.  You can **copy the embed code from there** and use it on your website.  It totally works, and you get an always up-to-date Run In Postman button.

For example, the Run In Postman embed code that's generated by the Postman app looks like this: 
Notice the ID (`5cc98b61c53d7f4f58b5`), which references a specific version of the collection.

```html
<div class="postman-run-button" data-postman-action="collection/import" data-postman-var-1="5cc98b61c53d7f4f58b5"></div>
````

But on the Postman Documenter website, the Run In Postman button for the same collection looks like this:
Notice the ID (`305204-599ad617-39ec-4c17-9d86-1be548c646ed-SW7XbA6V`), which references the collection itself, not a specific version.  Also, the second parameter "latest" specifically targets the latest version.

```html
<div class="postman-run-button" data-postman-action="collection/import" data-postman-var-1="305204-599ad617-39ec-4c17-9d86-1be548c646ed-SW7XbA6V" data-postman-var-2="latest"></div>
```

**Comment by preethammavin** [comment]
We have enabled two things which can solve your use case
1) Public Workpsaces which can have collections which are always up to date -where users can go fork the collections and the URL doesn't change unless you delete the said collection or workspace
2) We also the ability to embed the live collection link (which is derived from the public workspaces). This helps you embed that collections similar to static link - and enable forking from the RIP button. Added advantage is when you update the collections the consumers are also notified that the parent collection has been updated, and they can pull changes.
